gno/p/dao_interfaces/modules.gno

```diff
@@ -37,3 +37,17 @@ type IProposalModule interface {
 }
 
 type ProposalModuleFactory func(core IDAOCore) IProposalModule
+
+type IRolesModule interface {
```

Comment:
IMHO, it could be nice and more practice if we handle AddressOrName instead of only Address

Reply:
I disagree, if we do we have to add a way to resolve users by passing a resolver in the role module, increasing gas cost, dependencies and complexity
then if at some point usernames can change, we get a lot of headache
IMO it's better to always operate on addresses where possible